admin/app.js

```diff
@@ -19,9 +17,10 @@ const session = require('express-session')
 const MongoStore = require('connect-mongo')(session)
 const breadcrumbs = require('express-breadcrumbs')
 const fs = require('fs')
+const config = require('./config')
 
 mongoose.promise = global.Promise
-const connectionString = process.env.MONGO_CONNECTION_STRING || 'mongodb://localhost/mtc'
+const connectionString = config.MONGO_CONNECTION_STRING || 'mongodb://localhost/mtc'
```

Comment:
I would have thought it would make sense to default MONGO_CONNECTION_STRING to localhost in the config, rather than in the calling code.  It could also be used in the migration config.